[2022-02-03 16:35:05,081] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 16:35:05,098] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 16:35:05,099] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 16:35:05,099] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 16:35:05,099] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 16:35:05,116] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-03 16:35:05,122] {standard_task_runner.py:52} INFO - Started process 431 to run task
[2022-02-03 16:35:05,126] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpq9e4rcor', '--error-file', '/tmp/tmpy5i5sqz3']
[2022-02-03 16:35:05,127] {standard_task_runner.py:77} INFO - Job 15: Subtask format_to_parquet_task
[2022-02-03 16:35:05,220] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 16:35:05,286] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 16:35:05,321] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-03 16:35:17,000] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 16:35:17,050] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220203T163505, end_date=20220203T163517
[2022-02-03 16:35:17,310] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 16:35:17,354] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-03 16:47:57,852] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 16:47:57,865] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 16:47:57,866] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 16:47:57,866] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 16:47:57,866] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 16:47:57,886] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-03 16:47:57,893] {standard_task_runner.py:52} INFO - Started process 1045 to run task
[2022-02-03 16:47:57,901] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpnpqahv9r', '--error-file', '/tmp/tmp9r53iegr']
[2022-02-03 16:47:57,902] {standard_task_runner.py:77} INFO - Job 25: Subtask format_to_parquet_task
[2022-02-03 16:47:57,990] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 16:47:58,034] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 16:47:58,060] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-03 16:48:14,231] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 16:48:14,292] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220203T164757, end_date=20220203T164814
[2022-02-03 16:48:14,738] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 16:48:14,807] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-03 17:00:21,910] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 17:00:21,924] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 17:00:21,924] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:00:21,924] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 17:00:21,925] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:00:21,937] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-03 17:00:21,944] {standard_task_runner.py:52} INFO - Started process 1632 to run task
[2022-02-03 17:00:21,947] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpkjgq3l_j', '--error-file', '/tmp/tmpqtajcs_w']
[2022-02-03 17:00:21,948] {standard_task_runner.py:77} INFO - Job 39: Subtask format_to_parquet_task
[2022-02-03 17:00:22,012] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 17:00:22,054] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 17:00:22,072] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-03 17:00:35,747] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 17:00:35,761] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220203T170021, end_date=20220203T170035
[2022-02-03 17:00:35,990] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 17:00:36,059] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-03 17:15:36,483] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 17:15:36,496] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 17:15:36,497] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:15:36,497] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 17:15:36,497] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:15:36,510] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-03 17:15:36,516] {standard_task_runner.py:52} INFO - Started process 2316 to run task
[2022-02-03 17:15:36,520] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmphrpr6ij3', '--error-file', '/tmp/tmpfcjab4jn']
[2022-02-03 17:15:36,521] {standard_task_runner.py:77} INFO - Job 55: Subtask format_to_parquet_task
[2022-02-03 17:15:36,581] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 17:15:36,624] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 17:15:36,643] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-03 17:15:45,622] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 17:15:45,640] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220203T171536, end_date=20220203T171545
[2022-02-03 17:15:45,902] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 17:15:45,959] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 23:41:45,325] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-19 23:41:45,372] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-19 23:41:45,372] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:41:45,373] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-19 23:41:45,373] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:41:45,403] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-19 23:41:45,439] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpa1iyx8an', '--error-file', '/tmp/tmptb9ir7gn']
[2022-02-19 23:41:45,441] {standard_task_runner.py:77} INFO - Job 231: Subtask format_to_parquet_task
[2022-02-19 23:41:45,435] {standard_task_runner.py:52} INFO - Started process 470 to run task
[2022-02-19 23:41:45,537] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 812dd7ecce85
[2022-02-19 23:41:45,635] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 23:41:45,716] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-19 23:41:48,817] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/yellow_taxi_gcs_dag.py", line 30, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 867, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 876, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 99, in pyarrow.lib.check_status
pyarrow.lib.ArrowInvalid: CSV parse error: Expected 6 columns, got 18: 1,2019-03-01 01:15:31,2019-03-01 01:29:24,1,1.40,1,N,158,79,1,10,3,0.5,4.1,0,0.3,17.9,2.5
[2022-02-19 23:41:48,853] {taskinstance.py:1277} INFO - Marking task as UP_FOR_RETRY. dag_id=yellow_taxi_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220219T234145, end_date=20220219T234148
[2022-02-19 23:41:48,882] {standard_task_runner.py:92} ERROR - Failed to execute job 231 for task format_to_parquet_task
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/yellow_taxi_gcs_dag.py", line 30, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 867, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 876, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 99, in pyarrow.lib.check_status
pyarrow.lib.ArrowInvalid: CSV parse error: Expected 6 columns, got 18: 1,2019-03-01 01:15:31,2019-03-01 01:29:24,1,1.40,1,N,158,79,1,10,3,0.5,4.1,0,0.3,17.9,2.5
[2022-02-19 23:41:48,927] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-02-19 23:41:49,010] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
